src/components/book.js

```diff
@@ -1,12 +1,37 @@
 import PropTypes from 'prop-types';
+import style from './Book.module.css';
+import ProgressBar from './ProgressBar';
+
+const line = {
+  width: '5em',
+  border: '1px solid #e3e3e3',
+  transform: 'rotate(90deg)',
+  marginRight: '-4em',
+};
 
 export default function Book({ title, author, removeBook }) {
+  const chapter = () => Math.floor(Math.random() * 30);
   return (
-    <div className="wrapper">
-      <div>
-        <h2>{title}</h2>
-        <p>{author}</p>
-        <button type="button" onClick={removeBook}>Remove</button>
+    <div className={style.wrapper}>
+      <div className={style.book}>
+        <div className={style.book_details}>
+          <h2>{title}</h2>
+          <p>{author}</p>
+        </div>
+        <div className={style.btn_container}>
+          <button type="button" className={style.btn}>Comments</button>
+          <button type="button" onClick={removeBook} className={style.btn}>Remove</button>
+          <button type="button" className={style.btn}>Edit</button>
+        </div>
+      </div>
+      <ProgressBar />
+      <div style={line} />
+      <div className={style.chapter}>
+        <div>
+          <p className={style.chapter_heading}>CURRENT CHAPTER</p>
+          <p className={style.chapter}>{`Chapter ${chapter()}`}</p>
+        </div>
```

Comment:
- [ ] The progress bar of all books is not aligned as you can see below, due to the titles being a different length for each book, you can fix this by giving the book title container a relative width.
![afbeelding](https://github.com/tamoorsaeed22/book-store-react/assets/39240820/bfcedfbc-343b-4ba6-9216-1f90fd1a8625)

Reply:
@mikethreels hey mike the titles are a different length for each book, I have fixed the issue so they are aligned now .